python/tk_multi_loader/dialog.py

```diff
@@ -678,12 +780,12 @@ def _on_home_clicked(self):
 
                     # now see if our context object also exists in the tree of this profile
                     model = self._entity_presets[p].model
-                    if not model._cache_loaded:
-                        #items in model are not not loaded yet, create callback and wait for it
-                        if self._first_home_click.get('call_from_load', False):
-                            self._first_home_click['model'] = model
-                            model.data_refreshed.connect(self.functionCB)
-                            return
+#                     if not model._cache_loaded:
```

Comment:
Cleanup or at least add a comment that _cache_loaded was deprecated alongside the ShotgunOverlayModel from tk-framework-shotgunutils
https://support.shotgunsoftware.com/entries/95443067-Shotgun-Utils-Framework#toc_release_notes